tests/ui/test_display.py

```
@@ -0,0 +1,93 @@
+        self.assertIsInstance(printed_panel, Panel)
+        
+        # Check panel content
+        printed_table = printed_panel.renderable
+        self.assertIsInstance(printed_table, Table)
+        self.assertEqual(len(printed_table.rows), 3)
+        self.assertEqual(printed_table.rows[0].cells[1], 'Article 1')
+        self.assertEqual(printed_table.rows[1].cells[1], 'Article 2')
+        self.assertEqual(printed_table.rows[2].cells[1], 'Article 3')
+
+    @patch('src.ui.display.console')
+    def test_display_feeds_special_characters(self, mock_console):
```

Ruff (0.8.0)

Missing return type annotation for public function `test_display_feeds_special_characters`

Add return type annotation: `None`

(ANN201)